src/Comment.jsx

```diff
@@ -229,61 +238,74 @@ class _Comment extends React.Component {
             } else if ( this.state.editMode ) {
                 classes = 'mentions';
                 let completeEdit = <div onClick={this.completeEditing.bind(this)}
-                                        className={'fa fa-check mentions-ok'}/>;
+                    className={'fa fa-check mentions-ok'}/>;
                 let cancelEdit = <div onClick={this.cancelEditing.bind(this)}
-                                      className={'fa fa-undo mentions-cancel'}/>;
-                editMenu = (<div>
-                    {(this.state.editedComment !== null && this.state.editedComment !== this.state.comment)
-                        ? completeEdit
-                        : null}
-                    {cancelEdit}
-                </div>);
+                    className={'fa fa-undo mentions-cancel'}/>;
```

Comment:
I have picked react-fontawesome for ease of use - each icon is a React component. I was also drawn to it by its flexibility. For example, it is straightforward to quickly scale or recolor an icon. It might be that the `className` method provides similar benefits, and I'm all for converging on a single way of using font-awesome.

Regarding Cimpress icons, the ones shown here: https://static.ux.cimpress.io/assets/icons/index.html didn't suit my use case, so I used fontawesome. I noticed that UX asks to contact them in case an icon isn't displayed there. I'll try doing that and write back.

Reply:
👍